borgbackup/borg
# Bundle msgpack

trying to bundle msgpack 0.5.6 into borg, based on patches @FelixSchwarz used for Fedora (thanks!).

TODO:
- tests
- try to make it switchable, similar to how compression libs are handled (update: thinking about that, it might be unneccessary for borg 1.1.x because the msgpack 0.5.6 we will have bundled is the latest and greatest that is supported by borg anyway, so there is not much point in taking it from the system).
- check master branch msgpack adapter for the path structure, so it could be maybe adopted for master branch also.
